floodviz/views.py

```diff
@@ -105,8 +113,13 @@ def home():
 # -i [geodata field to use as id]
 # -c [css file]
 # -b [bounds]
-command = "svgis draw floodviz/static/data/gages.json floodviz/static/data/cities.json -o " \
-          "floodviz/static/data/sitemap.svg -f 500 -j epsg:2794 " \
-          "-i \"name\" -c floodviz/static/css/site_style.css -b " + \
-          str(bound1) + " " + str(bound2) + " " + str(bound3) + " " + str(bound4)
-os.system(command)
+# command = "svgis draw floodviz/static/data/gages.json floodviz/static/data/cities.json -o " \
+#           "floodviz/static/data/sitemap.svg -f 500 -j epsg:2794 " \
+#           "-i \"name\" -c floodviz/static/css/site_style.css -b " + \
+#           str(bound1) + " " + str(bound2) + " " + str(bound3) + " " + str(bound4)
+# os.system(command)
+
+x = svgis.map("floodviz/static/data/gages.json", scale=300, crs="epsg:2794", bounds=bounds)
```

Comment:
Actually that was pretty easy! But it still needs to happen when the view is rendered. I realize that it will slow down the rendering of the page, but that's why we will be freezing the app.